src/sql/parts/connection/common/connectionStore.ts

```diff
@@ -285,11 +286,18 @@ export class ConnectionStore {
 		return this.convertConfigValuesToConnectionProfiles(configValues);
 	}
 
-	public getProfileWithoutPassword(conn: IConnectionProfile): ConnectionProfile {
+	public getProfileWithoutPassword(conn: IConnectionProfile, removePasswordFromOptions: boolean = false): ConnectionProfile {
```

Comment:
why do we need a separate option to remove the password from options?  since the function name says "WithoutPassword" I would assume this option was assumed to be true?